### PR TITLE
parse variables as json, add a dedicated json var flag, close #43

### LINKED ISSUE #43: error in parsing json variables

### PR DESCRIPTION
**Pull request by shahidhk**

### CODE REVIEW

File: src/command.js [hunk 1]
```diff
@@ -181,10 +208,17 @@ GraphqurlCommand.flags = {
...
     char: 'v',
-    description: 'variables used in the query',
+    description: 'variables used in the query as key=value',
     multiple: true,
   }),
 
+  // variables for the query as JSON
+  variablesJSON: flags.string({
+    char: 'j',
+    description: 'variables used in the query as JSON key:value pairs',
```
Comment (wawhal) [inline src/command.js]: Description is not very clear. I suggest something along the lines of:

> query variables as a JSON string

File: src/command.js [hunk 2]
```diff
@@ -181,10 +208,17 @@ GraphqurlCommand.flags = {
   // variables for the query
   variable: flags.string({
     char: 'v',
-    description: 'variables used in the query',
+    description: 'variables used in the query as key=value',
```
Comment (wawhal) [inline src/command.js]: > query variables as key=value

File: src/command.js (hunk 1 above)
Comment (wawhal) [reply src/command.js]: It'd be great if you could also change the description of `--variablesFile` to say that a JSON file is expected

Comment (AnnieAtHasura) [reply src/command.js]: "query variables as a JSON string"  not clear 2 me, how about an example (not just a string either, nested json object)

**Pull request merged**